tests/RequestTest.php

```diff
@@ -1033,6 +1033,92 @@ public function testGetRefererInvalid()
         $this->assertNull($request->getReferer());
     }
 
+    /**
+     * @expectedException        Brick\Http\Exception\HttpBadRequestException
+     * @expectedExceptionMessage Invalid protocol: invalid_protocol
+     */
+    public function testGetCurrentShouldReturnHttpBadRequestException()
+    {
+        $request = new Request();
+        $_SERVER['SERVER_PROTOCOL'] = 'invalid_protocol';
+        $request->getCurrent();
+    }
+
+    public function testSetFiles()
+    {
+        $request = new Request();
+        $result = $request->setFiles(['./uploaded_file']);
+        $this->assertInstanceOf(Request::class, $result);
+        $this->assertContains('./uploaded_file', $request->getFiles());
+    }
+
+    public function testSetCookiesShouldRemoveCookieHeader()
+    {
+        $request = new Request();
+        $result = $request->setCookies([]);
+        $this->assertCount(0, $result->getCookie());
+    }
+
+    public function testSetRequestUriWithNoQueryString()
+    {
+        $request = new Request();
+        $result = $request->setRequestUri('http://localhost?');
+        $this->assertInstanceOf(Request::class, $result);
+        $this->assertContains('http://localhost?', $result->getRequestUri());
+    }
+
+    public function testGetAcceptShouldReturnEmptyArray()
+    {
+        $request = new Request();
+        $this->assertCount(0, $request->setUrl('http://localhost:8000')->getAccept());
```

Comment:
Instead of checking just the empty case, could you please replicate [the test for getAcceptLanguage()](https://github.com/brick/http/blob/master/tests/RequestTest.php#L1036-L1066), replacing the language tags with real-life `Accept:` header values, such as `*/*`, `text/html`, `image/jpeg`, ...